public/javascripts/data/user.js

```diff
@@ -2,24 +2,25 @@ const Database = require('better-sqlite3');
 const hashing = require('../hashing')
 
 
-
+// Function checks if user inputted username is available
 exports.signUp = (username) => {
     
     const db = new Database('public/javascripts/data/stories.db');
     const userStmt = db.prepare('SELECT user_name FROM users WHERE user_name = (?)');
     const userNameValue = userStmt.get(username);
 
     // Compare username to database to see if it's available
-    if(userNameValue == undefined) {
+    if(userNameValue != undefined) {
         db.close();
-        return true
+        return false;
     };
 
     db.close();
-    return false;
+    return true
```

Comment:
I'd replace all of the above with:

```js
let db = new Database('public/javascripts/data/stories.db');
let userStmt = db.prepare('SELECT user_name FROM users WHERE user_name = (?)');
let userName = userStmt.get(username);
db.close();
return userName;
```

Prefer let over const in this instance - These are not values which are guaranteed to never change

You can just return the result of `userName` since if there is no value found it returns `undefined`. Therefore, any caller of this method will be performing a check like:

```js
if (userName) do { ... }
```

And you don't need to force it into a `true`/`false` boolean type

Reply:
& don't forget to update the query to be case insensitive!